storage/storagepb/kv.proto

```diff
@@ -35,6 +35,10 @@ message Event {
   // a delete/expire event contains the previous
   // key-value
   KeyValue kv = 2;
-  // watchID is the ID of watching this event is sent to.
-  int64 watchID = 3;
+}
+
+message WatchResponse {
```

Comment:
we do not need this proto. we should use the WatchResponse proto defined in rpc.proto in etcdserverpb pkg.